Cesium.Runtime/StdIoFunctions.cs

```diff
@@ -24,12 +24,12 @@ public static void PutS(byte* str)
         }
     }
 
-    public static void PrintF(byte* str, void* varargs)
+    public static int PrintF(byte* str, void* varargs)
     {
         var formatString = Unmarshal(str);
         if (formatString == null)
         {
-            return;
+            return 1;
```

Comment:
According to C17 standard,
> The `printf` function returns the number of characters transmitted, or a negative value if an output or encoding error occurred.
```suggestion
            return -1;
```